### PR TITLE
Create poi

### PR DESCRIPTION
**Pull request by 0figueiredo**
-Get back new poi after it was created to display on the map and zoom on it.

### CODE REVIEW

File: routes/pointofinterest/route.js [hunk 1]
```diff
@@ -79,7 +80,13 @@ router.post('/', (req, res) => {
             if (errorGrades) {
               res.status(500).send(`Erreur lors de l'ajout des notes : ${errorGrades}`);
             } else {
-              res.sendStatus(200);
+              connection.query(getPoiInfosById, [resultId.poi_id], (er, re) => {
+                if (er) {
+                  res.status(500).send(`Erreur lors de la récupération du poi : ${e}`);
```
Comment (erabin05) [inline routes/pointofinterest/route.js]: 'e' is not define

File: routes/pointofinterest/route.js [hunk 2]
```diff
@@ -79,7 +80,13 @@ router.post('/', (req, res) => {
             if (errorGrades) {
               res.status(500).send(`Erreur lors de l'ajout des notes : ${errorGrades}`);
             } else {
-              res.sendStatus(200);
+              connection.query(getPoiInfosById, [resultId.poi_id], (er, re) => {
```
Comment (erabin05) [inline routes/pointofinterest/route.js]: not great arguments names

**Review by erabin05 — CHANGES_REQUESTED** [review]
- change arguments names in last connection

File: routes/pointofinterest/route.js (hunk 1 above)
Comment (0figueiredo) [reply routes/pointofinterest/route.js]: corrections done

File: routes/pointofinterest/route.js (hunk 2 above)
Comment (0figueiredo) [reply routes/pointofinterest/route.js]: correction done

**Review by erabin05 — APPROVED** [review]
Ok

**Pull request merged**